examples/11.ArduinoISP/ArduinoISP/ArduinoISP.ino

```diff
@@ -3,6 +3,9 @@
 // If you require a license, see
 // https://opensource.org/licenses/bsd-license.php
 //
+// Please note that this sketch still uses MISO/MOSI instead of CIPO/COPI.
```

Comment:
```suggestion
// Note that this sketch refers to the SPI bus pins using the legacy MISO/MOSI
// names rather than the modern CIPO/COPI names.
```

I feel the previous wording might give the reader the impression that there is a technical significance (vs. it being purely semantic). I think the suggested wording is more clear.

Reply:
Yeah, that makes sense. I changed it :)